sympy/geometry/ellipse.py

```diff
@@ -465,6 +465,50 @@ def evolute(self, x='x', y='y'):
         t2 = (self.vradius*(y - self.center.y))**Rational(2, 3)
         return t1 + t2 - (self.hradius**2 - self.vradius**2)**Rational(2, 3)
 
+    def director_circle(self, x='x', y='y'):
+        """Returns the equation for the director circle of an ellipse.
+
+        Raises :
+        =============
+        NotImplementedError : When the number of arguments is less than what is expected.
+
+        References :
+        =============
+        https://en.wikipedia.org/wiki/Director_circle
+
+        Input Parameters :
+        =============
+
+        str x : Default value is 'x'. Default argument, need not be given.
+        str y : Default value is 'y'. Default argument, need not be given.
+
+        Returns :
+        =============
+        A sympy expression for director circle of the defined ellipse.
+
+        Usage :
+        =============
+        Example 1 :
+        >>> from sympy import Ellipse, Point
+        >>> ellipse = Ellipse(Point(0, 0), 5, 1)
+        >>> ellipse.director_circle
+        >>> x**2 + y**2 - 26
+
+        Example 2:
+        >>> from sympy import Ellipse, Point
+        >>> ellipse = Ellipse(Point(2, 3), 2, 3)
+        >>> ellipse.director_circle
+        >>> (x - 2)**2 + (y - 3)**2 - 13
+        """
+        if (self.args) != 3:
```

Comment:
What is the purpose of this? Note that `self.args` is a tuple. It is never equal to the integer 3.

Reply:
Also the length of the tuple `self.args` is always equal to 3, so that need not be checked.

Reply:
what is the use of self.args @certik @jksuom

Reply:
Yes, that's my mistake. I'll correct it. I wanted to make sure that 3 arguments are supplied as inputs to the method but by default it is true. 
It should be len(self.args) != 3